clients/symfony-client.rst

```diff
@@ -24,7 +24,7 @@ to install one as well. `Nyholm PSR-7`_ is supported natively:
 Usage
 -----
 
-    use Symfony\Component\HttpClient\HttplugClient;
+    use Symfony\\Component\\HttpClient\\HttplugClient;
```

Comment:
The real issue here is that the code is not part of a code block. The following should solve root cause:

```rst
.. code-block:: php

    use Symfony\Component\HttpClient\HttplugClient;

    $symfonyClient = new HttplugClient();
```

Reply:
Ah, fixed, thank you.